components/list/list.css

```diff
@@ -0,0 +1,69 @@
+.list_group {
+    width: 15rem;
+}
+
+.list_section {
+    gap: 2rem;
+}
+
+.list_group_item {
+    position: relative;
+    padding: 0.75rem 1.25rem;
+    margin-bottom: -1px;
+    background-color: var(--white);
+    border: 1px solid #00000020;
```

Comment:
Define a variable for a color and use that

Reply:
Nope. I guess its not needed. Variables should be created only if the value is repeated or used in some other places.